appveyor.yml

```diff
@@ -39,24 +39,24 @@ install:
 
     # Add path, activate `conda` and update conda.
     - cmd: call %CONDA_INSTALL_LOCN%\Scripts\activate.bat
-    - cmd: conda update --yes --quiet conda
+    - cmd: conda.exe update --yes --quiet conda
```

Comment:
only question - what's a reason for it?

Reply:
This commit is made by the conda-forge admin. It is just for updating the testing for Windows platform and to update to latest version of Conda for testing.